### PR TITLE
Start testing against Kubernetes 1.24

### PR DESCRIPTION
**Pull request by swenson**

### CODE REVIEW

File: .github/workflows/acceptance.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kind-k8s-version: [1.16.15, 1.20.15, 1.21.10, 1.22.7, 1.23.4]
+        kind-k8s-version: [1.16.15, 1.20.15, 1.21.10, 1.22.7, 1.23.4, 1.24.1]
```
Comment (tvoran) [inline .github/workflows/acceptance.yaml:14]: Might as well update the other patch versions too:
```suggestion
        kind-k8s-version: [1.16.15, 1.20.15, 1.21.12, 1.22.9, 1.23.6, 1.24.1]
```

File: .github/workflows/acceptance.yaml [hunk 2]
```diff
@@ -23,6 +23,7 @@ jobs:
         with:
           config: test/kind/config.yaml
           node_image: kindest/node:v${{ matrix.kind-k8s-version }}
+          version: v0.14.0
 
       # Skip CSI tests if K8s version < 1.16.x
```
Comment (tvoran) [inline .github/workflows/acceptance.yaml:28]: If you want, we could also remove the < 1.16.x check step here (and the `check_skip_csi` function and calls in test/acceptance/csi.bats.

**Review by tvoran — APPROVED** [review]

File: .github/workflows/acceptance.yaml (hunk 2 above)
Comment (swenson) [reply .github/workflows/acceptance.yaml:28]: :+1:

**Comment by swenson** [comment]
Thanks!

**Pull request merged**